tools/hypergraph_statistics.cc

```diff
@@ -49,7 +49,7 @@ int main(int argc, char* argv[]) {
 
   kahypar::io::readHypergraphFile(graph_filename, num_hypernodes, num_hyperedges,
                                   index_vector, edge_vector, &hyperedge_weights, &hypernode_weights);
-  Hypergraph hypergraph(num_hypernodes, num_hyperedges, index_vector, edge_vector);
+  Hypergraph hypergraph(num_hypernodes, num_hyperedges, index_vector, edge_vector, 2, hyperedge_weights, hypernode_weights);
```

Comment:
Just for the sake of completeness, could you please also extend this to include the same statistics for hyperedge weights?

Reply:
Done